src/App.js

```diff
@@ -1,23 +1,32 @@
-import logo from './logo.svg';
 import './App.css';
+import Title from "./components/title/Title";
```

Comment:
it is better keep simple path, e.g. import { Title, Counter, ... } from "./components;

Reply:
Done